### ISSUE TITLE
Firefox extension breaks olx.pl login page

### ISSUE THREAD

**Issue by emikaadeo-git** [strongbox-password-safe/browser-autofill#7]
Steps to reproduce:
1. Go to [https://www.olx.pl](https://www.olx.pl) website
2. Click on "Your account"

![03_login_page](https://github.com/strongbox-password-safe/browser-autofill/assets/56483380/191d277f-5871-418f-b75c-29c0f498b86e)
3. Click on "E-mail" field
![01_login_page](https://github.com/strongbox-password-safe/browser-autofill/assets/56483380/b81af5ec-89ce-4652-a5ec-4d3b164a2ad1)

4. Website crashes and looks like below (it's possible to enter login credentials but impossible to click on "Log in" button)

![02_login_page](https://github.com/strongbox-password-safe/browser-autofill/assets/56483380/cb19bccc-cf5a-42a8-9da1-3457adf61f9f)

5. Disable Strongbox extension in Firefox
6. Repeat steps 1-3
7. Everything is ok

**Comment by strongbox-mark** [comment]
Super strange, we'll investigate... You can disable Inline Menus for this domain only in the settings tab, so this won't occur in the meantime.

**Comment by emikaadeo-git** [comment]
Hi,
just tried latest version 1.0.8 -> still no luck.
Any news about this issue?

BTW
Firefox shows that I have version 1.0.8 but the extension says is 1.0.7

<img width="175" alt="Zrzut ekranu 2023-08-21 o 22 07 28" src="https://github.com/strongbox-password-safe/browser-autofill/assets/56483380/cc99fe35-2708-44ab-b5ee-c8adccc3b556">

<img width="669" alt="Zrzut ekranu 2023-08-21 o 22 08 46" src="https://github.com/strongbox-password-safe/browser-autofill/assets/56483380/147478f9-7e63-462e-90aa-09bb8390d099">

**Comment by strongbox-mark** [comment]
We have something under development that I believe should fix this issue, hopefully we'll get it released this week... So look out for 1.0.9

**Comment by emikaadeo-git** [comment]
Just installed 1.0.9
The issue is fixed partially, website no longer brakes but when I do autofill, the `Log in` button is greyed out and impossible to click.
![04](https://github.com/strongbox-password-safe/browser-autofill/assets/56483380/28a9ddac-7445-4139-bcd9-a7c5b5416a94)

**Comment by strongbox-mark** [comment]
Yes, can see the same issue here. Good to hear it's nearly fixed... Will try to investigate shortly...

**Comment by strongbox-mark** [comment]
Hi @emikaadeo-git - Yeah we tried to find an easy fix for this and it doesn't look so simple... We also don't have many other similar issues on other sites so we think this might be some funky JS on that site... Can't say this is a high priority at the moment to be honest, but we'll keep it on the radar. Glad you like the other improvements.
